app/puppet-db/status.go

```diff
@@ -0,0 +1,43 @@
+package app
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/puppetlabs/puppetdb-cli/api/client/operations"
+	"github.com/puppetlabs/puppetdb-cli/json"
+	"github.com/puppetlabs/puppetdb-cli/log"
+
+	httptransport "github.com/go-openapi/runtime/client"
+)
+
+//GetStatusWithErrorDetails will check code manager api status and add error details to the error text
+func (puppetDb *PuppetDb) GetStatusWithErrorDetails() (*operations.GetStatusOK, error) {
+	resp, err := puppetDb.getStatus()
+
+	if err != nil {
+		if du, ok := err.(*operations.GetStatusDefault); ok {
+			if du.Payload != nil {
+				log.Debug(err.Error())
+				err = fmt.Errorf("[GET /status][%v] %v\n%v", du.Code(), du.Payload.Msg, json.PrettyPrintPayload(du.Payload.Details))
+			}
+		}
+	}
+	return resp, err
+}
+
```

Comment:
This might not be needed anymore

Reply:
I remember the change was needed to show error details similar to previous implementation, this might not be the case for puppetdb